lib/fog/kubevirt/compute/requests/get_server.rb

```diff
@@ -22,6 +23,15 @@ def populate_runtime_info(vm)
         rescue
           # do nothing if vmi doesn't exist
         end
+
+        def populate_runtime_nets(vm, server)
+          vmi = get_vminstance(vm[:metadata][:name])
```

Comment:
We already fetch vm instance here: https://github.com/fog/fog-kubevirt/pull/122/files#diff-d786e739ce6ac8a0ad6760f8a76d0adbR7 so we can spare another fetch.

In addition, would we always prefer vmi attributes on top of vm definition attributes ?

Reply:
VMI attributes are representing current state so I want to rely on them.

Will reuse VMI.